docs/userguide/Clients.md

```diff
@@ -1,149 +1,5 @@
-	'username' => 'foo',
-	'password' => 'bar'
-), array(
-	'logger' => $log,
-));
-```
-
-## 2. Authentication
-
-The Client does not automatically authenticate against the API on object creation - it waits for an API call. When this happens, it checks whether the current "token" has expired, and (re-)authenticates if necessary.
-
-You can force authentication, by calling:
-
-```php 
-$client->authenticate();
-```
-
-If the credentials are incorrect, a `401` error will be returned. If credentials are correct, a `200` status is returned with your Service Catalog.
-
-## 3. Service Catalog
-
-The Service Catalog is returned on successful authentication, and is composed of all the different API services available to the current tenant. All of this functionality is encapsulated in the `Catalog` object, which allows you greater control and interactivity.
-
-```php
-/** @var OpenCloud\Common\Service\Catalog */
-$catalog = $client->getCatalog();
-
-// Return a list of OpenCloud\Common\Service\CatalogItem objects
-foreach ($catalog->getItems() as $catalogItem) {
-	
-    $name = $catalogItem->getName();
-    $type = $catalogItem->getType();
-    
-    if ($name == 'cloudServersOpenStack' && $type == 'compute') {
-    	break;
-    }
-    
-    // Array of OpenCloud\Common\Service\Endpoint objects
-    $endpoints = $catalogItem->getEndpoints();
-    foreach ($endpoints as $endpoint) {
-    	if ($endpoint->getRegion() == 'DFW') {
-        	echo $endpoint->getPublicUrl();
-        }
-    }
-}
-```
-
-As you can see, you have access to each Service's name, type and list of endpoints. Each endpoint provides access to the specific region, along with its public and private endpoint URLs.
-
-## 4. Default HTTP headers
-
-To set default HTTP headers:
-
-```php
-$client->setDefaultOption('headers/X-Custom-Header', 'FooBar');
-```
-
-## User agents
-
-php-opencloud will send a default `User-Agent` header for every HTTP request, unless a custom value is provided by the end-user. The default header will be in this format:
-
-> User-Agent: OpenCloud/xxx cURL/yyy PHP/zzz
-
-where `xxx` is the current version of the SDK, `yyy` is the current version of cURL, and `zzz` is the current PHP version. To override this default, you must run:
-
-```php
-$client->setUserAgent('MyCustomUserAgent');
-```
-
-which will result in:
-
-> User-Agent: MyCustomUserAgent
-
-If you want to set a _prefix_ for the user agent, but retain the default `User-Agent` as a suffix, you must run:
-
-```php
-$client->setUserAgent('MyPrefix', true);
-```
-
-which will result in:
-
-> User-Agent: MyPrefix OpenCloud/xxx cURL/yyy PHP/zzz
-
-where `$client` is an instance of `OpenCloud\OpenStack` or `OpenCloud\Rackspace`.
-
-## 5. Other functionality
-
-For a full list of functionality provided by Guzzle, please consult the [official documentation](http://docs.guzzlephp.org/en/latest/http-client/client.html).
+https://doc.php-opencloud.com/en/latest/index.html
```

Comment:
I agree that a separate section on setting up OS/RS clients is not useful any more. There are, however, other parts of this document - specifically the parts about User Agents and Loggers - that are not included anywhere else in the new documentation. Do you think we ought to create a separate document (or documents) for them, if we are going to remove this document on Clients?